src/pages/ProfessorSurvey.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Radio,
+  RadioGroup,
+  Select,
+  Slider,
+  SliderFilledTrack,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  Textarea,
+  Tooltip,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { gql, useMutation } from "@apollo/client";
+import { useLoginStore } from "../stores/login";
+
+// these schemas will probably change later, all just example data
+const SUBMIT = gql`
+  mutation Login($username: String!, $password: String!) {
+    login(username: $username, password: $password) {
+      username
+      email
+      roles
+    }
+  }
+`;
+
+export const ProfessorSurvey = () => {
+  const [smallText, setSmallText] = useState("");
+  const [largeText, setLargeText] = useState("");
+  const [radioValue, setRadioValue] = useState("");
+  const [selectValue, setSelectValue] = useState("");
+  const [sliderValue, setSliderValue] = useState(100);
+  const [showTooltip, setShowTooltip] = React.useState(false);
+  const [submit, { data, loading, error }] = useMutation(SUBMIT);
+
+  const loginState = useLoginStore();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (loginState.loggedIn) {
+      navigate("/");
+    }
+  }, [loginState.loggedIn, navigate]);
+
+  useEffect(() => {
+    if (data && !error && !loading) {
+      // TODO: this route will change to whatever the default page is once the
+      // user is logged in
+      navigate("/");
+    }
+  }, [data, error, loading, navigate]);
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    submit({ variables: { smallText, radioValue, sliderValue } });
+    e.preventDefault();
+  };
+
+  return (
+    <Flex
+      w="100%"
+      minH="calc(100vh - 5.5rem)"
+      pt={30}
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+    >
+      <Container mb={32}>
```

Comment:
One thing you might want to do here is increase the container width using the `maxW="container.lg"` prop since this form has some more complex fields compared to the login page